src/__experimental__/components/textbox/textbox.d.ts

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-export interface TextboxProps {
+import { SpacingProps} from '../../../utils/helpers/options-helper';
```

Comment:
```suggestion
import { SpacingProps } from '../../../utils/helpers/options-helper'; 
```

Reply:
Need to get prettier running over ts files! You don't have to change this here

Reply:
I wonder why ts-lint didnt pick this up either.